clinical/src/main/kotlin/com/hartwig/actin/clinical/feed/emc/questionnaire/QuestionnaireExtraction.kt

```diff
@@ -11,52 +11,66 @@ object QuestionnaireExtraction {
+            val whoStatus = QuestionnaireCuration.toWHO(entry.subject, value(lines, mapping[QuestionnaireKey.WHO_STATUS]))
+            val infectionStatus =
+                QuestionnaireCuration.toInfectionStatus(value(lines, mapping[QuestionnaireKey.SIGNIFICANT_CURRENT_INFECTION]))
+            val ecg = QuestionnaireCuration.toEcg(value(lines, mapping[QuestionnaireKey.SIGNIFICANT_ABERRATION_LATEST_ECG]))
+            val stage = QuestionnaireCuration.toStage(entry.subject, value(lines, mapping[QuestionnaireKey.STAGE]))
+            val questionnaire = Questionnaire(
+                date = entry.authored,
+                tumorLocation = value(lines, mapping[QuestionnaireKey.PRIMARY_TUMOR_LOCATION]),
+                tumorType = if (QuestionnaireVersion.version(entry) == QuestionnaireVersion.V0_1) "Unknown" else value(
+                    lines,
+                    mapping[QuestionnaireKey.PRIMARY_TUMOR_TYPE]
+                ),
+                biopsyLocation = value(lines, mapping[QuestionnaireKey.BIOPSY_LOCATION]),
+                stage = stage.curated,
+                treatmentHistoryCurrentTumor = toList(value(lines, mapping[QuestionnaireKey.TREATMENT_HISTORY_CURRENT_TUMOR])),
+                otherOncologicalHistory = toList(value(lines, mapping[QuestionnaireKey.OTHER_ONCOLOGICAL_HISTORY])),
+                secondaryPrimaries = secondaryPrimaries(lines, mapping[QuestionnaireKey.SECONDARY_PRIMARY]),
+                nonOncologicalHistory = toList(value(lines, mapping[QuestionnaireKey.NON_ONCOLOGICAL_HISTORY])),
+                ihcTestResults = value(lines, mapping[QuestionnaireKey.IHC_TEST_RESULTS])?.let { listOf(trimWhitespaceAndCommas(it)) },
+                pdl1TestResults = toList(value(lines, mapping[QuestionnaireKey.PDL1_TEST_RESULTS])),
+                hasMeasurableDisease = hasMeasurableDisease.curated,
+                hasBrainLesions = brainLesionData.curated?.present(),
+                hasActiveBrainLesions = brainLesionData.curated?.active(),
+                hasCnsLesions = cnsLesionData.curated?.present(),
+                hasActiveCnsLesions = cnsLesionData.curated?.active(),
+                hasBoneLesions = hasBoneLesions.curated,
+                hasLiverLesions = hasLiverLesions.curated,
+                otherLesions = otherLesions(entry, lines, mapping),
+                whoStatus = whoStatus.curated,
+                unresolvedToxicities = toList(value(lines, mapping[QuestionnaireKey.UNRESOLVED_TOXICITIES])),
+                infectionStatus = infectionStatus.curated,
+                ecg = ecg.curated,
+                complications = toList(value(lines, mapping[QuestionnaireKey.COMPLICATIONS])),
+            )
+            val errors = listOf(brainLesionData, cnsLesionData, hasBoneLesions, hasLiverLesions, hasMeasurableDisease, whoStatus, infectionStatus, ecg, stage).flatMap(
+                ValidatedQuestionnaireCuration<*>::errors
+            )
```

Comment:
I would move `flatMap` to the next line:
```suggestion
            val errors = listOf(brainLesionData, cnsLesionData, hasBoneLesions, hasLiverLesions, hasMeasurableDisease, whoStatus, infectionStatus, ecg, stage)
                .flatMap(ValidatedQuestionnaireCuration<*>::errors)
```